product-mini/platforms/linux/CMakeLists.txt

```diff
@@ -155,7 +155,7 @@ set_target_properties (iwasm PROPERTIES POSITION_INDEPENDENT_CODE ON)
 
 install (TARGETS iwasm DESTINATION bin)
 
-target_link_libraries (iwasm vmlib ${LLVM_AVAILABLE_LIBS} ${UV_A_LIBS} -lm -ldl -lpthread)
+target_link_libraries (iwasm vmlib ${LLVM_AVAILABLE_LIBS} ${UV_A_LIBS} ${TENSORFLOW_LIB} -lm -ldl -lpthread)
```

Comment:
Considering that wasi-nn will support other than Tensorflow in the future, isn't NN_LIBS better as a MACRO name?

Reply:
good catch!

Reply:
https://github.com/bytecodealliance/wasm-micro-runtime/pull/2308/commits/25f593a342d9e004635f6ccd303bfbcaaa90509b